Buildings/Controls/OBC/Utilities/PIDWithAutotuning/Relay/Controller.mo

```diff
@@ -75,18 +76,33 @@ equation
          color={0,0,127}));
   connect(greMeaSet.y, swi.u2)
     annotation (Line(points={{22,0},{58,0}},color={255,0,255}));
-  connect(conErr.u2, u_s) annotation (Line(points={{-62,14},{-90,14},{-90,0},{-120,
-          0}}, color={0,0,127}));
   connect(swi1.u3, u_s) annotation (Line(points={{-62,-58},{-90,-58},{-90,0},{-120,
           0}}, color={0,0,127}));
-  connect(swi1.y, greMeaSet.u) annotation (Line(points={{-38,-50},{-20,-50},{-20,
-          -6},{-2,-6}}, color={0,0,127}));
-  connect(swi1.y, conErr.u1) annotation (Line(points={{-38,-50},{-20,-50},{-20,-6},
-          {-70,-6},{-70,26},{-62,26}}, color={0,0,127}));
+
   connect(trigger, swi1.u2) annotation (Line(points={{-80,-120},{-80,-50},{-62,-50}},
         color={255,0,255}));
   connect(u_m, swi1.u1) annotation (Line(points={{0,-120},{0,-80},{-70,-80},{-70,
           -42},{-62,-42}}, color={0,0,127}));
+   connect(swi1.y, conErr.u1) annotation (Line(points={{-38,-50},{-20,-50},{-20,-6},
+          {-70,-6},{-70,26},{-62,26}}, color={0,0,127}));
+   connect(conErr.u2, u_s) annotation (Line(points={{-62,14},{-90,14},{-90,0},{-120,
+          0}}, color={0,0,127}));
+  if reverseActing then
```

Comment:
should not have the `if else` in the equation section. We only allow the `connect` statement.

To enable or disable the connection, you can add a `CDL.Reals.MultiplyByParameter` instance and make the instance conditional. This will enable/disable the connection.

Reply:
I made two components for comparing the reference and the measurement, which are enabled/disenabled based on <code>reverseActing</code> and removed the <code>if else</code> in the equation section.